apps/server/src/modules/instance/entity/instance.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, Property } from '@mikro-orm/core';
+import { BaseEntityWithTimestamps } from '@shared/domain/entity/base.entity';
```

Comment:
```suggestion
import { BaseEntityWithTimestamps } from '@shared/domain/entity';
```

Reply:
This does not work because the index.ts exports ALL_ENTITES which leads to dependency cycles